packages/web/src/features/register-club/frame/RegisterClubTypeSelectionFrame.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+
+import styled from "styled-components";
+
+import Button from "@sparcs-clubs/web/common/components/Button";
+import FlexWrapper from "@sparcs-clubs/web/common/components/FlexWrapper";
+import Info from "@sparcs-clubs/web/common/components/Info";
+import PageHead from "@sparcs-clubs/web/common/components/PageHead";
+import ClubButton from "@sparcs-clubs/web/features/register-club/components/ClubButton";
+
+const ClubButtonWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 20px;
+  align-self: stretch;
+`;
+
+const RegisterClubTypeSelectionFrame = () => {
+  enum RegistrationType {
+    renewalRegistration = "renewalRegistration",
+    promotionalRegistration = "promotionalRegistration",
+    provisionalRegistration = "provisionalRegistration",
+    unselected = "unselected",
+  }
+
+  const [selectedType, setSelectedType] = useState<RegistrationType>(
+    RegistrationType.unselected,
+  );
```

Comment:
이 경우에는 unselected type을 만드는 것보다는 이런 식으로 useState의 타입에 null 을 추가하는 것이 나을듯!
```suggestion
  const [selectedType, setSelectedType] = useState<RegistrationType | null>(null);
```